### PR TITLE
9998: refactor deprecate consolidatedCasesGroupCountMap

### PR DESCRIPTION
**Pull request by mmarcotte**

### CODE REVIEW

File: shared/src/business/useCases/judgeActivityReport/getCasesByStatusAndByJudgeInteractor.test.ts [hunk 1]
```diff
@@ -86,204 +91,26 @@ describe('getCasesByStatusAndByJudgeInteractor', () => {
...
-
-    applicationContext
-      .getPersistenceGateway()
-      .getCasesMetadataByLeadDocketNumber.mockResolvedValueOnce(
-        casesForLeadDocketNumber,
-      );
-
-    const result = await getCasesByStatusAndByJudgeInteractor(
+  it('calls getDocketNumbersByStatusAndByJudge with excludeMemberCases flag = true (stripping out the consolidated member case)', async () => {
+    await getCasesByStatusAndByJudgeInteractor(
```
Comment (ujahio) [inline shared/src/business/useCases/judgeActivityReport/getCasesByStatusAndByJudgeInteractor.test.ts:95]: 👏🏿 👏🏿

File: web-api/src/persistence/elasticsearch/getCountOfConsolidedCases.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+          term: {
+            'leadDocketNumber.S': leadDocketNumber,
+          },
+        },
+      },
+      index: 'efcms-case',
+    },
+  });
+
+  console.log(results);
```
Comment (ujahio) [inline web-api/src/persistence/elasticsearch/getCountOfConsolidedCases.ts]: nit: Remove log

File: web-api/src/persistence/elasticsearch/getDocketNumbersByStatusAndByJudge.test.ts [hunk 3]
```diff
@@ -17,14 +17,18 @@ describe('getDocketNumbersByStatusAndByJudge', () => {
...
   };
 
-  const responseResults = [
-    { docketNumber: '11315-18' },
-    { docketNumber: { S: '11316-18' } },
-  ];
+  const standaloneCase = { docketNumber: '11315-18' };
+  const leadCase = { docketNumber: '11316-18', leadDocketNumber: '11316-18' };
+  const memberCase = { docketNumber: '11317-18', leadDocketNumber: '11316-18' };
+  const responseResults = [standaloneCase, leadCase, memberCase];
```
Comment (ujahio) [inline web-api/src/persistence/elasticsearch/getDocketNumbersByStatusAndByJudge.test.ts:23]: 👏🏿 👏🏿 
Separating the docket numbers into prescriptive variable names really does help.

**Review by ujahio — APPROVED** [review]

**Comment by mmarcotte** [comment]
will become a part of flexion#10049

**Review by TomElliottFlexion — APPROVED** [review]

**Pull request merged**